aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -15,41 +15,44 @@ module aptos_framework::fungible_asset {
     const EAMOUNT_CANNOT_BE_ZERO: u64 = 1;
     /// The transfer ref and the fungible asset do not match.
     const ETRANSFER_REF_AND_FUNGIBLE_ASSET_MISMATCH: u64 = 2;
-    /// Account cannot transfer or receive fungible assets.
+    /// Store is disabled from sending and receiving this fungible asset.
     const EUNGATED_TRANSFER_IS_NOT_ALLOWED: u64 = 3;
     /// Insufficient balance to withdraw or transfer.
     const EINSUFFICIENT_BALANCE: u64 = 4;
     /// The fungible asset's supply has exceeded maximum.
     const EMAX_SUPPLY_EXCEEDED: u64 = 5;
-    /// More tokens than remaining supply are being burnt.
+    /// Cannot burn more tokens than the remaining supply.
     const ESUPPLY_UNDERFLOW: u64 = 6;
     /// The mint ref and the the store do not match.
     const EMINT_REF_AND_STORE_MISMATCH: u64 = 7;
     /// Account is not the store's owner.
     const ENOT_STORE_OWNER: u64 = 8;
     /// Transfer ref and store do not match.
     const ETRANSFER_REF_AND_STORE_MISMATCH: u64 = 9;
-    /// Burn ref and store do not match.
+    /// Burn ref and store do not match addresses.
```

Comment:
Addresses is confusing. How about “are not for the same fungible asset address”?